tests/fixtures/data/json/resource_export_business_data_truth.json

```diff
@@ -41,6 +42,12 @@
             "data": {
               "677f236e-09cc-11e7-8ff7-6c4008b05c4c": null
             },
+            "nodegroup": {
+              "cardinality": "1",
+              "legacygroupid": null,
+              "nodegroupid": "677f236e-09cc-11e7-8ff7-6c4008b05c4c",
+              "parentnodegroup_id": null
+            },
```

Comment:
@apeters 

This was fun, it took a while to track down. 

It's here: https://github.com/archesproject/arches/blob/dev/7.6.x/arches/app/utils/betterJSONSerializer.py#L209

Essentially because `nodegroup` is now a property on `TileModel` it gets serialized since `TileModel` doesn't override the default `serialize` method.

To fix this, I added `TileModel.serialize()` with a default exclude of `['nodegroup']`